engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
I like the idea of leaving a safety valve in the engine although primarily relying on the backend. Someone (I think it was @mcovarr) made the point on a different topic a week or so ago that we're going to have to keep bulletproofing from craptastic backends in mind from here on out.